decidim-proposals/lib/decidim/proposals/test/factories.rb

```diff
@@ -223,6 +223,14 @@
       end
     end
 
+    trait :with_attachments_and_card_image_allowed do
```

Comment:
the name of this trait is not true, it is not adding attachments.. should we rename it?
```suggestion
    trait :with_card_image_allowed do
```